src/node_env_var.cc

```diff
@@ -389,13 +390,33 @@ static void EnvEnumerator(const PropertyCallbackInfo<Array>& info) {
       env->env_vars()->Enumerate(env->isolate()));
 }
 
+static void EnvDefiner(Local<Name> property,
+                       const PropertyDescriptor& desc,
+                       const PropertyCallbackInfo<Value>& info) {
+  Environment* env = Environment::GetCurrent(info);
+  if (desc.has_get() || desc.has_set() || desc.has_configurable() ||
+      desc.has_enumerable() || desc.has_writable()) {
+    env->ThrowError("Cannot specify data or accessor descriptors"
+                    " on 'process.env'");
+  } else if (desc.has_value()) {
```

Comment:
Do we need the `if (desc.has_value())` check at all? Should the value not be present in all other cases? Or should we maybe throw in case it's not set?

Reply:
if coder only call `defineProperty` without `value`, do not throw an error is the good way I think🤔

Reply:
and use `else` would be a good way, I'll handle it